docs/core/introduction.md

```diff
@@ -6,404 +6,52 @@ ms.date: 02/24/2022
 ms.custom: "updateeachrelease"
 recommendations: false
 ---
-# What is .NET? Introduction and overview
+# What is .NET?
 
-.NET is a free, open-source development platform for building many kinds of apps, such as:
+NET is a free, cross-platform, open source developer platform for building many different types of [applications](tutorials/with-visual-studio-code.md). .NET is built on a [high-performance runtime](https://devblogs.microsoft.com/dotnet/category/performance/) that is used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/app/). It is built with the community on [GitHub](https://github.com/dotnet/core).
 
-* [Web apps, web APIs, and microservices](/aspnet/core/introduction-to-aspnet-core#recommended-learning-path)
-* [Serverless functions in the cloud](/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-csharp)
-* [Cloud native apps](../architecture/cloud-native/index.md)
-* [Mobile apps](https://dotnet.microsoft.com/learn/xamarin/hello-world-tutorial/intro)
-* Desktop apps
-  * [Windows WPF](/dotnet/desktop/wpf/)
-  * [Windows Forms](/dotnet/desktop/winforms/)
-  * [Universal Windows Platform (UWP)](/windows/uwp/get-started/create-a-hello-world-app-xaml-universal)
-* [Games](https://dotnet.microsoft.com/apps/games)
-* [Internet of Things (IoT)](../iot/index.yml)
-* [Machine learning](../machine-learning/index.yml)
-* [Console apps](tutorials/with-visual-studio-code.md)
-* [Windows services](/aspnet/core/host-and-deploy/windows-service)
+You can use .NET for client apps, for [desktop](/dotnet/desktop/), [mobile](/xamarin/), [browser](aspnet/core/blazor), and [games](https://dotnet.microsoft.com/apps/games).
 
-Share functionality among different apps and app types by using [class libraries](../standard/class-libraries.md).
+You can use .NET for [cloud apps](../architecture/cloud-native/index.md), [Web apps and APIs](/aspnet/core/introduction-to-aspnet-core#recommended-learning-path), and [serverless functions](/azure/azure-functions/functions-create-first-function-vs-code?pivots=programming-language-csharp). .NET cloud apps are competitively measured by the [techempower benchmark](https://www.techempower.com/benchmarks/#test=composite).
 
-With .NET, your code and project files look and feel the same no matter which type of app you're building. You have access to the same runtime, API, and language capabilities with each app.
+.NET can be used for many other apps, like [IoT](dotnet/iot/), [Tools](/dotnet/core/tools/global-tools), [machine learning](../machine-learning/index.yml), and [NativeAOT](https://github.com/dotnet/runtime/issues/61231).
```

Comment:
should we add NativeAOT after it launches? We are also not talking about MAUI here, although we should add it once it ships.

Reply:
```suggestion
.NET can be used for many other apps, like [IoT](dotnet/iot/), [Tools](/dotnet/core/tools/global-tools), and [machine learning](../machine-learning/index.yml).
```